src/Umbraco.Web/PropertyEditors/ContentPickerPropertyEditor.cs

```diff
@@ -33,17 +33,12 @@ protected override PreValueEditor CreatePreValueEditor()
 
         internal class ContentPickerPreValueEditor : PreValueEditor
         {
-            [PreValueField("showOpenButton", "Show open button", "boolean")]
+            [PreValueField("showOpenButton", "Show open button (this feature is in preview!)", "boolean", Description = " Opens the node in a dialog")]

```

Comment:
Can't do it without quite a lot of work with regards to attributed, leave it for now and create a task to revisit later